webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/InterpolatedValueCompiler.scala

```diff
@@ -397,18 +406,18 @@ object AnalyseNodeInterpolation extends AnalyseInterpolation[ConfigValue, Interp
   }
 }
 
-class InterpolatedValueCompilerImpl extends InterpolatedValueCompiler {
+class InterpolatedValueCompilerImpl(propertyEngineService: PropertyEngineService) extends InterpolatedValueCompiler {
```

Comment:
you correctly pass it as an argument here, but it should not be used in `parse` method but in the implementation of `expandWithPropertyEngine(context: I, engineName: String (= e), nameSpace: List[String] (= m), params: ConfigValue (= p)): IOResult[String]`